pyproject.toml

```diff
@@ -31,6 +31,8 @@ dependencies = [
     "wandb",
     "scipy",
     "datasets",
+    "pandas",
+    "pydicom",
```

Comment:
I would move pydicom as an optional dependency, similar to [here](https://github.com/deepinv/deepinv/blob/7c282240a8088a56e304708731516a2132ac1946/deepinv/models/bm3d.py#L6)

Reply:
Under a list of optional dependencies for everything related to datasets ?

Reply:
yes good idea